ui/src/components/facets/FacetHeader.js

```diff
@@ -29,27 +29,47 @@ const styles = {
     fontWeight: 600,
     padding: "11px 18px 0 0",
     textAlign: "right"
+  },
+  removeFacet: {
+    padding: "7px 0 0 50px"
   }
 };
 
 class FacetHeader extends Component {
+  state = {
```

Comment:
Usually state is [defined in constructor](https://reactjs.org/docs/state-and-lifecycle.html#adding-local-state-to-a-class).